app/src/main/java/com/buzzardparking/buzzard/states/LeavingState.java

```diff
@@ -20,12 +20,22 @@ public LeavingState(Context context, PlaceManager placeManager, CameraManager ca
 
     @Override
     public void start() {
+        if (isReady() || isReadyCache()) {
```

Comment:
this is a check for if it is the first time initializing the app or if  it is just moving from one screen to the next. Either way we need to make sure the resources are there